### ISSUE TITLE
Crash report from Dynamo 2.3.0.5885

### ISSUE THREAD

**Issue by GuyC64** [DynamoDS/Dynamo#11284]
# Issue Description
Please fill in the following information to help us reproduce the issue:

### What did you do?
(Fill in here)

### What did you expect to see?
(Fill in here)

### What did you see instead?
(Fill in here)

### What packages or external references (if any) were used?
(Fill in here)

### Stack Trace
```
(From the Dynamo crash window select 'Details' -> 'Copy' and paste here)
```

---
OS: `Microsoft Windows NT 10.0.19042.0`
CLR: `4.0.30319.42000`
Dynamo: `2.3.0.5885`

**Comment by QilongTang** [comment]
@GuyC64 Hi, would you fill in more details so our team could digest your crash?

**Comment by Amoursol** [comment]
Hello @GuyC64  - Please reopen with the template filled in with additional information. Unfortunately as it stands there is not enough for us to further action your report. We would be looking for an uploaded .DYN file, screenshots and/or reproducible steps.

**Comment by GuyC64** [comment]
Hi, Sorry for my late return. The crash seems to be happening because of my VPN connection. After discussing with the IT department everything went back to normal. Thank's

________________________________
De : Sol Amour <notifications@github.com>
Envoyé : 4 décembre 2020 11:11
À : DynamoDS/Dynamo <Dynamo@noreply.github.com>
Cc : GuyC64 <guyc64@hotmail.com>; Mention <mention@noreply.github.com>
Objet : Re: [DynamoDS/Dynamo] Crash report from Dynamo 2.3.0.5885 (#11284)


Closed #11284<https://github.com/DynamoDS/Dynamo/issues/11284>.

**Comment by Amoursol** [comment]
Brilliant! Happy to hear that you have managed to figure it out on your end 😊 Glad Dynamo is back up and running!
